CONTRIBUTING.md

```diff
@@ -38,6 +38,9 @@ whether as part of the official Superset docs,
 in docstrings, `docs/*.rst` or even on the web as blog posts or
 articles.
 
+To build the docs, simply run ``./build.sh`` from the ``docs/`` folder.
```

Comment:
@mistercrunch, I tried running this and it fails:

```bash
$ cd docs
$ pip install -r requirements.txt
$ ./build.sh
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.7.1

Exception occurred:
  File "/Users/bdealmeida/src/incubator-superset/env/lib/python3.6/site-packages/sphinxcontrib/youtube/youtube.py", line 3, in <module>
    import urlparse
ModuleNotFoundError: No module named 'urlparse'
The full traceback has been saved in /var/folders/x9/9gcx_mp522x88bjx3dr38b7r0000gn/T/sphinx-err-1uywj973.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 2
cp: _build/html: No such file or directory
```

Looks like it currently works only for Python 2.

Reply:
this is fixed by https://github.com/apache/incubator-superset/pull/5731 which removes the code that raises in py3 (extension='sphinxcontrib.youtube')